### PR TITLE
Adding Ideas

### PR DESCRIPTION
**Pull request by Aqsa48**
Adding idea into idea.md

### CODE REVIEW

File: .github/workflows/master_deploy.yml [hunk 1]
```diff
@@ -19,7 +19,8 @@ jobs:
               run: npm install
             - name: Deploy Application to GH Pages
               run: |
-                  git config user.email ${{ secrets.EMAIL }}
                   git config user.name ${{ secrets.USERNAME }}
```
Comment (DemonDaddy22) [inline .github/workflows/master_deploy.yml]: Why did you change this file? This shouldn't be changed, please revert your changes to this file.

File: ideas.md [hunk 2]
```diff
@@ -25,3 +25,12 @@ Moving Hamburger Animation is a good project to hold command on React.
...
 - [https://www.youtube.com/watch?v=dIyVTjJAkLw](https://www.youtube.com/watch?v=dIyVTjJAkLw)
 - [https://hamburger-react.netlify.app/](https://hamburger-react.netlify.app/)
+
+### Portfolio Creator
+Portfolio creator allows the user to create own portfolio by adding up their own data and later on by clicking download it can be also be  downloaded in pdf form.)
+<br>
+<strong>Links for references</strong>
+- [https://github.com/Aqsa48/Aqsa48-MyPortfolio](https://github.com/Aqsa48/Aqsa48-MyPortfolio)
+- [https://aqsa-portfolio.herokuapp.com/](https://aqsa-portfolio.herokuapp.com/)
+
```
Comment (DemonDaddy22) [inline ideas.md]: Remove these empty lines at the end, no need for them

**Comment by Aqsa48** [comment]
I hope It would work fine now.

**Review by DemonDaddy22 — APPROVED** [review]

**Comment by DemonDaddy22** [comment]
LGTM. I've approved it, will merge it by tomorrow.

**Comment by DemonDaddy22** [comment]
@all-contributors please add @Aqsa48 for doc
